### ISSUE TITLE
Segfault on binary operations

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#1669]
Originally reported in sass/node-sass#1107 (comment)

```
foo {
  bar: #{100%/3}
}
```
Ruby Sass / LibSass 3.2.5
```
foo {
  bar: 100%/3; }
```
LibSass 3.3
```
[1]    69839 segmentation fault
```
Spec added sass/sass-spec#577

**Comment by saper** [comment]
Fixed by #1668 ?

**Comment by xzyfer** [comment]
Should be
